install/alpine-caddy-install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: cobalt (cobaltgit)
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://caddyserver.com/
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Caddy"
+$STD apk add --no-cache caddy caddy-openrc
+msg_ok "Installed Caddy"
+
+read -r -p "${TAB3}Would you like to install xCaddy Addon? <y/N> " prompt
+if [[ "${prompt,,}" =~ ^(y|yes)$ ]]; then
+  GO_VERSION="$(curl -fsSL https://go.dev/VERSION?m=text | head -1 | cut -c3-)" setup_go
```

Comment:
```suggestion
  setup_go
```
`setup_go` without GO_VERSION always fetches the latest from the website. there is no need for you to parse the website